example/fedora/minimal-40-aarch64.yaml

```diff
@@ -36,16 +28,15 @@ otk.define:
       vfs_type: "ext4"
       path: "/boot"
       options: "defaults"
-    boot-efi:
+    boot_efi:
       uuid: "7B77-95E7"
       vfs_type: "vfat"
       path: "/boot/efi"
       options: "defaults,uid=0,gid=0,umask=077,shortname=winnt"
       passno: 2
 
 otk.target.osbuild.qcow2:
+  otk.include: "repository/${version}/repositories.yaml"
   pipelines:
-    - otk.include: "osbuild/buildroot.yaml"
-    - otk.include: "osbuild/pipeline/tree.yaml"
```

Comment:
Thanks for moving the `  otk.include: "repository/${version}/repositories.yaml"` out -  the other two inludes (`buildroot.yaml` and `tree.yaml`) we want to keep

Reply:
with those rows I got this error:
  File "/Users/michalgold/git/edge-management/image-builder-project/otk/src/otk/transform.py", line 39, in resolve
    return resolve_list(ctx, state, data)
  File "/Users/michalgold/git/edge-management/image-builder-project/otk/src/otk/transform.py", line 129, in resolve_list
    return [resolve(ctx, state, val) for val in tree]
  File "/Users/michalgold/git/edge-management/image-builder-project/otk/src/otk/transform.py", line 129, in <listcomp>
    return [resolve(ctx, state, val) for val in tree]
  File "/Users/michalgold/git/edge-management/image-builder-project/otk/src/otk/transform.py", line 37, in resolve
    return resolve_dict(ctx, state, data)
  File "/Users/michalgold/git/edge-management/image-builder-project/otk/src/otk/transform.py", line 116, in resolve_dict
    return resolve(ctx, state, call(key, resolve(ctx, state, val)))
  File "/Users/michalgold/git/edge-management/image-builder-project/otk/src/otk/transform.py", line 37, in resolve
    return resolve_dict(ctx, state, data)
  File "/Users/michalgold/git/edge-management/image-builder-project/otk/src/otk/transform.py", line 67, in resolve_dict
    val = substitute_vars(ctx, val)
  File "/Users/michalgold/git/edge-management/image-builder-project/otk/src/otk/tree.py", line 23, in wrapper
    return function(*args, **kwargs)
  File "/Users/michalgold/git/edge-management/image-builder-project/otk/src/otk/transform.py", line 285, in substitute_vars
    return ctx.variable(m.group("name"))
  File "/Users/michalgold/git/edge-management/image-builder-project/otk/src/otk/context.py", line 103, in variable
    raise TransformVariableLookupError(f"could not find {parts!r}")
otk.error.TransformVariableLookupError: could not find ['packages', 'keys']

Reply:
I see - thank you! Maybe just comment them out in this case? This way we will not forget about them :)